modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -207,6 +164,14 @@ res = requests.post(
 pretty(res)
 ```
 
+--
+rpk::
```

Comment:
Is it possible to put the rpk tab first than the curl and python?

Reply:
@r-vasquez definitely, I'll make that change